app/views/pages/trg/contacts.njk

```diff
@@ -95,4 +107,8 @@ any CCC questions or concerns in a timely manner to ensure no delays at TRG.</p>
 </div>
 
 https://hscic365.sharepoint.com/:w:/r/sites/TR&G/_layouts/15/Doc.aspx?sourcedoc=%7B1AD2E35C-F615-4FFF-AB89-CAE195195231%7D&file=TRG%20Entry%20Checklist.docx&action=default&mobileredirect=true
```

Comment:
Seems to be a rogue sharepoint link here

Reply:
I have made the content authors aware of this, it seems to have been added for a temporary reason and will be changed in the next round of content changes. I will leave this open for now for reference.